### ISSUE TITLE
Add release date to ember inspector

### ISSUE THREAD

**Issue by drewcovi** [alisdair/ember-cli-build-date#1]
can this info be added to ember inspector as is done with ember-cli-app-version? (perhaps i missed it) thanks!

**Comment by alisdair** [comment]
I don't think so, or at least I don't know how to do so cleanly. [ember-cli-app-version registers the app name as a library with a version](https://github.com/embersherpa/ember-cli-app-version/blob/master/addon/initializer-factory.js#L11), which I don't see any way to hook into.
